src/lib.rs

```diff
@@ -79,21 +79,21 @@
 //! embedded graphics code, or produce examples and interactive demos to show of embedded graphics
 //! features.
 //!
-//! ![It can display all sorts of embedded-graphics test code.](https://raw.githubusercontent.com/jamwaffles/embedded-graphics/master/assets/simulator-demo.png)
+//! ![It can display all sorts of embedded-graphics test code.](https://raw.githubusercontent.com/embedded-graphics/embedded-graphics/master/assets/simulator-demo.png)
 //!
 //! Take a look at the [simulator examples] to see what
 //! embedded-graphics can do, and how it might look on a display. You can run the examples like
 //! this:
 //!
 //! ```bash
-//! git clone https://github.com/jamwaffles/embedded-graphics.git
+//! git clone https://github.com/embedded-graphics/embedded-graphics.git
 //! cd embedded-graphics
 //!
 //! cargo run -p embedded-graphics-simulator --example hello
 //! ```
 //!
-//! [simulator]: https://github.com/jamwaffles/embedded-graphics/tree/master/simulator
-//! [simulator examples]: https://github.com/jamwaffles/embedded-graphics/tree/master/simulator/examples
+//! [simulator]: https://github.com/embedded-graphics/embedded-graphics/tree/master/simulator
+//! [simulator examples]: https://github.com/embedded-graphics/embedded-graphics/tree/master/simulator/examples
```

Comment:
This will need to be changed once we have moved the examples to the separate repo. Maybe we should add an issue for the examples move and add a note to remember to change this.
```suggestion
//! [simulator]: https://github.com/embedded-graphics/simulator
//! [simulator examples]: https://github.com/embedded-graphics/simulator/tree/master/examples
```

Reply:
I've opened embedded-graphics/examples#1 to track.